### ISSUE TITLE
[Jupiter benchmark] Deployment of test infrastructure

### ISSUE THREAD

**Issue by Sicofonia** [gravitee-io/issues#8195]
Based on the "Define APIM deployment architecture", deploy apim to be able to run load test on a gateway v3 without Jupiter.
It will be use to run full v3 api that involves no jupiter code at all.
Based on the "Define APIM deployment architecture", deploy apim to be able to run load test on a gateway with Jupiter enabled.
It will be use to run both jupiter and jupiter with V3 compatibility mode.
Deploy an echo api dedicated to apim performance tests
We don't want to use a mutualized api as it could negatively impact performances if multiple teams run tests in the same time.
Be able to run performance tests on the performance platform on demand.
Think about how to get credentials for k6 user to invke apim or am 
For september, we will go with 'on demand' only as scheduling is not mandatory to achieve the objectives to compare v3 and jupiter performances

### Tasks
- [x] Deploy APIM V3 on performance platform
- [x] Deploy APIM JUpiter on performance platform
- [ ] Deploy echo api
- [ ] Setup for run K6 on performance platform

### Acceptance Criteria
- All deployments of APIM, echo API done as per description
- Can run a k6 test in the newly deployed performance platform

**Comment by Sicofonia** [comment]
We need the task here to be done to unblock this one:
https://graviteedevops.atlassian.net/jira/software/projects/PERF/boards/17/backlog?selectedIssue=PERF-3

**Comment by phiz71** [comment]
Put this ticket "on hold" since async entrypoints have a higher priority.
I'll be back 😎

**Comment by phiz71** [comment]
According to a discussion I had with David 2 weeks ago, there have been some misunderstanding about where the performance platform should be deployed. Waiting for a future sync meeting with Mo and David.
